src/search.ts

```diff
@@ -5,29 +5,23 @@ import { NotImplementedError } from "./core/utilities/errors";
 
 type BlockNumber = number;
 
-/**
- * @typedef BaseFilters
```

Comment:
typedoc preset doesn't like this?

Reply:
yep, interface docs are actually discouraged in general, but if we want to have them, we can use labels on each key.